bin/deploy.sh

```diff
@@ -57,9 +57,7 @@ esac
 ABBREV_LENGTH=`node ${DIRNAME}/../tasks/config-abbrev --env=$ENV`
 ROLLBAR_ABBREV=`node ${DIRNAME}/../tasks/config-rollbar-abbrev --env=$ENV`
 COMMIT=`git rev-parse --short=${ABBREV_LENGTH} HEAD`
-COMMIT_ROLLBAR=`git rev-parse --short=${ROLLBAR_ABBREV} HEAD`
-BRANCH=`git rev-parse --abbrev-ref HEAD`
-REV="$COMMIT"
+BRANCH="${CI_COMMIT_REF_NAME:-`git rev-parse --abbrev-ref HEAD`}"
```

Comment:
I think we covered this in some other PR: you are introducing a dependency on CI solution here. The previous solution relies only on git. Until we move this script out of the public repo I would suggest to keep maximum compat.

Reply:
Feel free to fix it, but this is definitely better then slowing down our CI with unnecessary checkout. Since it has fallback, I guess it's good enough.